From 51ae1a968215244fb6b8a027d12b34b20d4d3022 Mon Sep 17 00:00:00 2001
From: Trio Patch Bot <patch-bot@users.noreply.github.com>
Date: Tue, 15 Sep 2026 00:05:27 +0200
Subject: [PATCH] feat: g7-reading-time-with-seconds

Trio-Patch-Source-Branch: feature/g7sensorkit-timestamp-seconds
Trio-Patch-Base: 1900ad76c43c42918e650a7863f7ae315bf689d2
Trio-Patch-Feature-Tip: 7fbf12120793e3c996c1a1784e536ba4b3038ff0
Trio-Patch-Generated: 2026-09-14T22:05:17Z
Trio-Patch-Source-Commits:
 fc24f3972964564b48d0f8cab9aedf851bcb9114 Update G7SensorKit (show reading timestamp with seconds)
 7fbf12120793e3c996c1a1784e536ba4b3038ff0 Merge dev (0.8.4.106) into feature/g7sensorkit-timestamp-seconds
Trio-Patch-Source-PatchIds:
 ed97df36e4b18886dc68346ea67ceb00ed61b72f
 5560470aadb1706fcb843cabf5981f33df1fe39d
---
 .gitmodules | 2 +-
 G7SensorKit | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 6978b165c..af35c2b05 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -12,7 +12,7 @@
 	url = https://github.com/loopandlearn/RileyLinkKit
 [submodule "G7SensorKit"]
 	path = G7SensorKit
-	url = https://github.com/loopandlearn/G7SensorKit.git
+	url = https://github.com/cachrisman/G7SensorKit.git
 [submodule "MinimedKit"]
 	path = MinimedKit
 	url = https://github.com/loopandlearn/MinimedKit.git
diff --git a/G7SensorKit b/G7SensorKit
index 1486241a9..0013c90d4 160000
--- a/G7SensorKit
+++ b/G7SensorKit
@@ -1 +1 @@
-Subproject commit 1486241a9e4a6f3da13d98542b8e39a02938ab96
+Subproject commit 0013c90d4e2f2a5f3c49c19096c58d01667f078c
-- 
2.55.0

